engine/reference/commandline/scout.md

```diff
@@ -10,4 +10,14 @@ in the source repository on GitHub:
 
 https://github.com/docker/scout-cli-plugin
 -->
+
+> **Early Access**
+>
+> Docker Scout is an [early access](../../../release-lifecycle.md#early-access-ea)
+> product, and requires a Docker Pro, Team, or Business subscription.
+>
+> If you're interested in Docker Scout for your organization and want to
+> learn more, get in touch by filling out the contact form on the
+> [Docker Scout product page](https://docker.com/products/docker-scout){: target="_blank" rel="noopener" }.
```

Comment:
Can we have an include for this to avoid dedup?

Reply:
Yeah, I was considering for a bit; it was already duplicated, and there was still some discussion if the "cli" and "scout" (as a whole) have the same stability (early access / beta), so I decided to (for now) just copy/paste.

Let me know if you feel strongly about that; I can update.

Reply:
Let me have a quick look at making a include